server/Main.hs

```diff
@@ -123,10 +112,7 @@ server externs initNamesEnv initEnv port = do
 
 getOpts :: Int -> Scotty.Options
 getOpts port = def
-  { settings =
-      Warp.setHost "127.0.0.1"
```

Comment:
Can you revert this change please? Each setting we’re overriding ought to have its own line here I think. I understand that the $ is unnecessary here but to me having each setting on its own line is more important.